inc/saibridge.h

```diff
@@ -420,6 +426,14 @@ typedef enum _sai_bridge_flood_control_type_t
     /** Flood on the L2MC group */
     SAI_BRIDGE_FLOOD_CONTROL_TYPE_L2MC_GROUP,
 
+    /**
+     * @brief Flood on all sub-ports and L2MC group
```

Comment:
What is the use case? Is the idea to create a bunch of snooped interfaces with few statically added for bcast/mcast?

Reply:
Using this API extension, you can easily add flooding to router (by adding a bridge port type router) or to a tunnel, on top of the regular behavior of flooding to all local ports, without needing to manage the list of local ports, by choosing the combined flood type.
A sample use case for flooding to router, is VXLAN L3 peering use case, demoed in OCP, where unknown unicast traffic is sent to router.